shim.go

```diff
@@ -96,6 +98,36 @@ func newShimConfig(config PodConfig) interface{} {
 	}
 }
 
+func shimRunning(pid int) (bool, error) {
+	process, err := os.FindProcess(pid)
+	if err != nil {
+		return false, err
+	}
+
+	if err := process.Signal(syscall.Signal(0)); err != nil {
+		return false, nil
+	}
+
+	return true, nil
+}
+
+func stopShim(pid int) error {
+	running, err := shimRunning(pid)
```

Comment:
@jodh-intel sorry but I am not clear on what could happen here and how to solve this ?